### PR TITLE
Translation caiquanyou

### PR DESCRIPTION
**Pull request by caiquanyou**

### CODE REVIEW

**Comment by MichaelChirico** [comment]
LGTM! Can you help confirm @Leo-Lee15?

File: po/R-zh_CN.po [hunk 1]
```diff
@@ -808,89 +808,94 @@ msgid "NA in 'new' at positions"
...
-msgstr ""
+msgstr "在'old' 中存在重复名称"
 
 msgid "'old' is type"
-msgstr ""
+msgstr "'old' 的类型是"
 
 msgid "but should be integer, double or character"
-msgstr ""
+msgstr "应该为整型,双整型或者字符型"
```
Comment (Leo-Lee15) [inline po/R-zh_CN.po]: `double` 翻译为“浮点型” 可能会更好。

File: po/R-zh_CN.po [hunk 2]
```diff
@@ -808,89 +808,94 @@ msgid "NA in 'new' at positions"
...
-msgstr ""
+msgstr "‘new’ 的长度为"
 
 msgid "NA (or out of bounds) in 'old' at positions"
-msgstr ""
+msgstr "NA(或超出界限)出现在'old' 的位置"
 
 msgid "of 'old' is '"
-msgstr ""
+msgstr "of 'old' 是"
```
Comment (Leo-Lee15) [inline po/R-zh_CN.po]: 错误信息很可能是 `NA (or out of bounds) in 'old' at positions * of 'old' is *`. 这里把`of 'old'是`这样翻译可能造成前后割裂。建议翻译成`是`。

File: po/R-zh_CN.po [hunk 3]
```diff
@@ -808,89 +808,94 @@ msgid "NA in 'new' at positions"
...
+msgstr "在列名中重复出现,仅第一个"
+"会改动，如下:"
 
 msgid "other items in old that are also duplicated in column names."
-msgstr ""
+msgstr "old中的其他项在列名中出现重复"
 
 msgid "Items of 'old' not found in column names:"
-msgstr ""
+msgstr "在列名中为找到 'old' 有关项"
```
Comment (Leo-Lee15) [inline po/R-zh_CN.po]: 应该是`列名中未找到'old`有关项'。

File: po/R-zh_CN.po [hunk 4]
```diff
@@ -808,89 +808,94 @@ msgid "NA in 'new' at positions"
...
 msgid "use.names=NA invalid"
-msgstr ""
+msgstr "use.names=NA 无效赋值"
 
 msgid ""
 "use.names='check' cannot be used explicitly because the value 'check' is new "
 "in v1.12.2 and subject to change. It is just meant to convey default "
 "behavior. See ?rbindlist."
-msgstr ""
+msgstr "use.names='check'无法明确使用，由于值'check'为新增"
```
Comment (Leo-Lee15) [inline po/R-zh_CN.po]: 个人觉得应该是'请勿直接使用use.names='check'，因为...'

File: po/zh_CN.po [hunk 5]
```diff
@@ -1511,157 +1511,165 @@ msgstr ""
...
 #, c-format
 msgid "Unable to allocate %s of contiguous virtual RAM. %s allocation."
-msgstr ""
+msgstr "无法分配 %s 的连续虚拟内存。 %s 已分配。"
 
 #: fread.c:442
 #, c-format
 msgid "Avoidable %.3f seconds. %s time to copy.\n"
-msgstr ""
+msgstr "可避免的 %.3f ，秒。 %s 复制用时\n"
```
Comment (Leo-Lee15) [inline po/zh_CN.po]: "秒"前面多了一个逗号

**Comment by caiquanyou** [comment]
thanks@Leo-Lee15

**Comment by MichaelChirico** [comment]
Hey @caiquanyou just noticed you made some updates after Guangzheng's review, is your part finished now?

**Comment by caiquanyou** [comment]
@MichaelChirico  yes,my part finished now

**Comment by MichaelChirico** [comment]
Great!! Thanks @caiquanyou and @Leo-Lee15

**Pull request merged**